libraries/engage/page/widgets/NestedCategoryFilter/NestedCategoryFilter.jsx

```diff
@@ -0,0 +1,139 @@
+import React, {
+  useCallback, useEffect, useMemo, useState,
+} from 'react';
+import { I18n, ButtonLink } from '@shopgate/engage/components';
+import { bin2hex } from '@shopgate/pwa-common/helpers/data';
```

Comment:
you still import a bunch of modules from the "old" libraries.  please replace the block till @virtuous/router with

```js
import { bin2hex } from '@shopgate/engage/core/helpers';
import { useRoute } from '@shopgate/engage/core/hooks';
import { CATEGORY_PATH } from '@shopgate/engage/category';
import { makeStyles } from '@shopgate/engage/styles';
import { themeConfig } from '@shopgate/engage';
```